src/app/components/map/aoi-dimensions/aoi-dimensions.component.html

```diff
@@ -11,7 +11,7 @@
                 {{ aoiEdition?.area | squareKm | formatNumber}}  km²
             </div>
             <div class="message" *ngIf="aoiEdition?.area === 0">
-                {{ aoiEdition?.areaMessage}}
+                {{ aoiEdition?.areaMessage | translate}}
```

Comment:
If the areaMessages are part of an enum or in the ts file, you can use marker to translate them so that they would be caught by the i18n:extract and i18n:init scripts to fill the internationalization files